packages/did-client/src/did-client.ts

```diff
@@ -32,104 +23,39 @@ interface VeridaSelfTransactionConfigPart  {
  */
 
 export interface DIDClientConfig {
-    // vda-did resolver - only
-    identifier : string // string value "vda:did:0x12...f" or ethereum address
-    chainName? : string 
-    chainId? : string | number
-
-    // common to vda-did resolver & vda-did self transaction config
-    provider? : Provider
-    rpcUrl? : string
-    registry?: string
-    web3?: any
-}
-
-// Interfaces for saving DID Document
-interface VerificationMethodBase {
-    id: string
-    type: string
-    controller: string
-}
-
-interface Controller extends VerificationMethodBase {
-    blockchainAccountId: string
-}
-
-/**
- * Interface to DIDClient instance
- */
-export interface DIDClient {
-
-    authenticate(
-        veridaPrivateKey: string,
-        callType: CallType,
-        web3Config: VeridaSelfTransactionConfigPart|VeridaMetaTransactionConfig
-    ): void;
-
-    getDid(): string | undefined;
-
-    getPublicKey(): string | undefined;
-
-    save(document: DIDDocument): Promise<boolean>;
-
-    get(did?: string): Promise<DIDDocument | null>
-    
+    network: string             // `testnet` OR `mainnet`
```

Comment:
Users can't create DIDClient with chain ID. It's now supporting for only mainnet or testnet.
Question : Verida will not support for other chains later? It will run on only Polygon network?
If we need to support another chains, we should update this configuration.

Reply:
Noted.

We don't plan to support other chains at this stage. If we did, we will add them to the `vda-did-resolver` list of supported networks.